pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml

```diff
@@ -882,6 +884,12 @@ spec:
                             - secret
                             type: object
                         type: object
+                        x-kubernetes-validations:
+                        - message: If a non-TCP protocol is used in any of the ports,
+                            then there should not be any HTTP rules set
+                          rule: '!has(self.rules) || !has(self.rules.http) || !self.ports.exists_one(p,
+                            !has(p.protocol) || p.protocol != ''TCP'')'
+                      maxItems: 32
```

Comment:
Curious to know why we chose `32` as the max value ?

Reply:
I go with different power of 2, and 32 seems to be the maximum that we can have to avoid cost budget error 

https://github.com/kubernetes/kubernetes/blob/dae1859c896d742de1ee60a349475f8e28b61995/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go#L1316